packages/merge-styles/src/Stylesheet.ts

```diff
@@ -63,6 +63,11 @@ export interface IStyleSheetConfig {
 }
 
 const STYLESHEET_SETTING = '__stylesheet__';
+/**
+ * MSIE 11 doesn't cascade styles based on DOM ordering, but rather on the order that each style node
+ * is created. As such, to maintain consistent priority, IE11 should reuse a single style node.
+ */
+const REUSE_STYLE_NODE = /rv:11.0/.test(navigator.userAgent);
```

Comment:
navigator may not exist in a server rendered environment. You should probably check that.

Reply:
Fixed.
```
PS E:\VSO\office-ui-fabric-react> node -p "typeof navigator !== 'undefined' && /rv:11.0/.test(navigator.userAgent)"
false
```